cmd/verrazzano-operator/main.go

```diff
@@ -66,7 +66,7 @@ func main() {
 
 	if startController {
 		// start the controller
-		if err = controller.Run(2); err != nil {
+		if err = controller.Run(2, kubeconfig); err != nil {
```

Comment:
kubeconfig is supposed to only be required out of cluster.  How does this work in-cluster

Reply:
i see that an empty string causes the code to look for in-cluster config so this looks ok